src/modules/auth/ForgetPasswordSuccess/ForgetPasswordSuccess.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { useRouter } from 'next/router';
+import { useTranslation } from 'react-i18next';
+import { GlobalRoute } from '@src/constants/PageRoutes';
+import { VStack, Text } from '@chakra-ui/react';
+import { Button } from '@common/components';
+
+export default function ForgetPasswordSuccess() {
+  const { t } = useTranslation('auth');
+
+  const router = useRouter();
+  const backToHome = () => router.push(GlobalRoute.Root);
```

Comment:
Here as it is not a super big function on the button's `onClick` event, we don't necessarily need to define the function of `backToHome` and call it in the button `onClick`, we can just write an arrow function in the `Button`, like this:
` <Button type="button" onClick={() => router.push(GlobalRoute.Root)}>`
which can simplify the code and easier to read through